lms/templates/dashboard/_dashboard_entitlement_unenrollment_modal.html

```diff
@@ -0,0 +1,33 @@
+<%page expression_filter="h"/>
+
+<%!
+from django.utils.translation import ugettext as _
+%>
+
+<div id="unenroll-entitlement-modal" class="entitlement-modal unenroll-entitlement-modal" aria-hidden="true">
+  <div class="entitlement-modal-inner-wrapper" role="dialog" aria-labelledby="unenroll-entitlement-modal-title">
```

Comment:
You'll likely want aria-modal="true" here too.

Reply:
Added.